### ISSUE TITLE
Windows nodes don't seem to show up

### ISSUE THREAD

**Issue by mnaamani** [paritytech/substrate-telemetry#116]

**Comment by mnaamani** [comment]
Going to try hosting a telemetry server and see if I can find the problem

**Comment by mnaamani** [comment]
Works like a charm on latest master branch. Perhaps website isn't running latest version?

**Comment by maciejhirsz** [comment]
No, it's the latest version that's running online. The default URL is `wss://telemetry.polkadot.io/submit/`.

**Comment by mnaamani** [comment]
@maciejhirsz Yes I'm using the correct URL. When running a node from linux or mac seems to work fine and node is listed in telemetry (official). But if running a node from windows machine, it does not show up.

**Comment by mnaamani** [comment]
Given that it worked on localhost telemetry server, maybe it could be an issue with SSL/TLS cert verification, or perhaps CORS configuration on telemetry.polkadot.io endpoint?

**Comment by maciejhirsz** [comment]
I'm just setting up a windows VM to test it out. SSL is my hunch, it usually is the culprit in those situations.
